tests/functional/test_delete.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from tests import BaseGeneralInterfaceTest
+from s3transfer.manager import TransferManager
+
+
+class TestDeleteObject(BaseGeneralInterfaceTest):
```

Comment:
I do not think anything needs to change here, but you had asked me earlier about this so I figured to make a note here. For the functional test, I usually add a general test where I take the default stubbed responses add an extra argument and make sure that gets passed in correctly, but it looks like you already have that in the unit test so it does not really need a functional test. The functional test is only nice because it guarantees that that specific test will always be there as interfaces can come and go with unit tests.